pom.xml

```diff
@@ -21,6 +21,7 @@
         <module>webconsole</module>
         <module>jsonlogging</module>
         <module>prometheus-webconsole</module>
+        <module>consul</module>
```

Comment:
This module transitively depends on guava 19.0. The json logging module already depends on guava 22.0. The orbitz consul thing does not work with guava 22.0. The solution for now should be to downgrade the guava version we use, put it in the parent pom and thus only manage the version once (possibly as a managed dependency), and make an upstream pull request for orbitz to work with guava 22.0.